configure.ac

```diff
@@ -0,0 +1,58 @@
+AC_CONFIG_MACRO_DIRS([m4])
+AC_INIT([awscli], [2.7.31])
```

Comment:
We should probably update this to the current version of v2 is now. It will automatically get bumped but it would be good to see it be in sync to start.